### ISSUE TITLE
ferenOS crashes suddenly

### ISSUE THREAD

**Issue by kagirohi** [feren-OS/Bug-Reporting-Center#171]
**Package Name or Section of Feren OS where this bug resides**
Where does this bug happen? (e.g.: feren-themer-cinnamon) ___
ferenOS
**Describe the bug**

ferenOS Suddenly crashes during I have used it recently. 
I can't control ferenOS.(I guess what especially KDE's process often have crashed) 
It was working normally in spring or summer,but It has often crashed Within 1 month.

**Steps to Reproduce**

1.It SUDDENLY crashes.I don't know when It craches.
2.Sometimes, It has not crash,I use.
**Expected behavior**

feren OS Freezes 
To be NOT able to Writting files in Dir,(Read only??)
To be NOT able to Starting Applications, 
Opening Vivaldi suddenly exits. 
**Screenshots**

**Additional context**

Maybe It's winter, so I suspect static electricity.But,Windows10 is working normally.
ferenOS classic is working normally too. 
I'm sorry for my bad English

**Comment by dominichayesferen** [comment]
Gonna need more information than that.

For example, what version of Feren OS is it?

**Comment by kagirohi** [comment]
I use FerenOS 2020.07

**Comment by kagirohi** [comment]
Operating System: Feren OS 2020.07
KDE Plasma Version: 5.19.4
KDE Frameworks Version: 5.72.0
Qt Version: 5.14.2
Kernel Version: 5.4.0-52-generic
OS Type: 64-bit
Processors: 4 × Intel® Core™ i5-6500 CPU @ 3.20GHz
Memory: 15.6 GiB of RAM
Graphics Processor: GeForce GTX 650 Ti/PCIe/SSE2

Pasting from KDE system information.

**Comment by kagirohi** [comment]
I force a reboot with a physical reset button whenever FerenOS crashes or freezes.
When I check the Bios while restarting, my ssd is not recognized on the BIOS.
But,Booting normally after I stop supplying electricity to my computer.my ssd is recognized on the BIOS
My ssd is "ADATA sp550".ferenOS is installed in this ssd.but, /home is installed other hdd.

Google translation is great!!

**Comment by dominichayesferen** [comment]
Have you recently went into a Live Session, opened Disks or GParted and checked the Feren OS partition on your hard drive?

**Comment by kagirohi** [comment]
I have NOT recently went into a Live session,opened Disks or gparted.

**Comment by dominichayesferen** [comment]
You ought to go into Disks in a Live Session, select the big Feren OS ext4 partition, hit the cog below the big bar you just clicked an item of, and then choose 'Check Filesystem...'.

**Comment by kagirohi** [comment]
I have not found errors.Filesystem is undamaged.

**Comment by kagirohi** [comment]
![Screenshot_20201115_checkdisk](https://user-images.githubusercontent.com/57948679/99200454-c4a55e80-27e8-11eb-93ad-4b1ca5564553.png)
I forget to say that I install windows10 and ferenOS(expect /home partition) with alongside on my ssd.(Dual boot)
/home partition is installed on hitachi hhd.

**Comment by kagirohi** [comment]
![Screenshot_20201115_checkdisk_home](https://user-images.githubusercontent.com/57948679/99200631-c28fcf80-27e9-11eb-8745-93e8f8315f33.png)
Filesystem is undamaged on this hdd too.

**Comment by dominichayesferen** [comment]
Hmm... so it's not that the filesystem is damaged... weird...

Did you let the Live Session you installed Feren OS in fully check itself before you went to install it? Did it say there were no errors found in that?

**Comment by kagirohi** [comment]
I don't remember that I let the Live session in fully check itself.
It is possible that my live session FerenOS installer did not have the "check media" feature.

**Comment by dominichayesferen** [comment]
Given how your system seems to be oddly crashing, it might be for the best to get the latest ISO and let its media check go on during boot... and then consider reinstalling with it.

**Comment by kagirohi** [comment]
Thank you.
I will consider reinstalling or changing ssd.

The other day, I received the following message from ferenOS in (tty1..6)
I recived the view of  message from ferenOS when ferenOS freezed in tty1's mode.

![IMG_20201121_232828](https://user-images.githubusercontent.com/57948679/99892177-8c54c300-2cb5-11eb-8685-bb8db5db6a82.jpg)

ferenOS(tty 1)

> systemd_journald[374]:Failed to write entry(... items ,...bytes)

> blk_update_request: I/O error, dev sda,sector .... op 0x0 ...........

**Comment by dominichayesferen** [comment]
Feren OS has mounted your system in read-only mode. This means that there's either issues detected on your Feren OS partition (bad blocks), or at worst the SSD itself.

Make sure to `sudo fsck -A -P -V` in Feren OS's Recovery Mode (accessible via GRUB -> Advanced Options for Feren OS) to fix any bad blocks on your partitions. (note: All the partitions your system has access to will be scanned for errors when running this command)

**Comment by kagirohi** [comment]
I confirmed.Thank you.
By the way,Your Youtube channel's icon is Japanese anime("Chuunibyou demo koigashitai!" in Japanese「中二病でも恋がしたい！」)?
Do you like it?
I have never seen it,but I have some funny pictures!
![DSCF1895](https://user-images.githubusercontent.com/57948679/100165758-6706c580-2efe-11eb-9449-29716450f5dc.JPG)

**Comment by kagirohi** [comment]
![DSCF1899](https://user-images.githubusercontent.com/57948679/100165820-93224680-2efe-11eb-9f00-3ffeaadc2fa7.JPG)

**Comment by kagirohi** [comment]
![DSCF1901](https://user-images.githubusercontent.com/57948679/100165853-a1706280-2efe-11eb-828c-679a48dde7a4.JPG)
![DSCF1911](https://user-images.githubusercontent.com/57948679/100165873-b1884200-2efe-11eb-953a-be1e9a59d211.JPG)
![DSCF1914](https://user-images.githubusercontent.com/57948679/100165931-d977a580-2efe-11eb-9be0-fc10553a6e30.JPG)

If FerenOS will keeps crashing,I can't access these pictures...(well..It's a joke,I have had backup already,don't worry.)

**Comment by dominichayesferen** [comment]
Off-topic, but YES I really like that anime... partially because I can relate a ton to Rikka in the Season 1 ending scene and the build-up to that ending scene.

**Comment by dominichayesferen** [comment]
Back to on-topic: Did you manage to run that command I suggested you run? If so, has the issue mostly, if not fully, resolved itself?

**Comment by kagirohi** [comment]
I ran that command,but it is not resolved.
So,I cloned system another device.
My ssd(ADATA SP550) may have something wrong.
Do you think  that it will be resolved if I install FerenOS 2020.11 instead of FerenOS 2020.07 or Installing FerenOS 2020.07  at another filesystem. ( for example ,Btrfs or Ntfs ...)

**Comment by dominichayesferen** [comment]
I wouldn't think the base change that occured in Feren OS November 2020 would affect anything. It mounted it as read-only for a reason, afterall.

**Comment by kagirohi** [comment]
Thank you for your advice.
I tried installing and using FerenOS 2020.11 at Btrfs on SSD(ADATA SP550),but it crashed and freezed in using.
I thought ext4 filesystem has something wrong with this SSD,but having misunderstood.
Your advice are right.
This SSD has badblocks,it made FerenOS crashed.
 I cloned system of FerenOS another device,be using system Comfortable now.

Thank you,I wish you a good year!
